app/models/concerns/itemizable.rb

```diff
@@ -130,6 +130,19 @@ def line_item_items_quantity_is_positive
     end
   end
 
+  def line_item_items_quantity_is_at_least(threshold)
+    return if storage_location.nil?
+
+    line_items.each do |line_item|
+      next unless line_item.item
+      next if line_item.quantity.nil? || line_item.quantity >= threshold
+
+      errors.add(:inventory,
+                "#{line_item.item.name}'s quantity " \
+                "needs to be above #{threshold}")
```

Comment:
Should be "needs to be at least #{threshold}"

Reply:
fixed!